src/xcode/ENA/ENA/Source/Scenes/DynamicTableViewController/DynamicTableViewController.swift

```diff
@@ -293,25 +293,32 @@ private extension UITableViewCell {
 
 	func addSeparator(_ location: SeparatorLocation) {
 		let separator = UIView(frame: bounds)
-		contentView.addSubview(separator)
+
+		let seperatorView: UIView
+		if let existingSeperatorView = self.backgroundView {
+			seperatorView = existingSeperatorView
+		} else {
+			seperatorView = UIView()
+			self.backgroundView = seperatorView
+		}
+		seperatorView.addSubview(separator)
```

Comment:
I still think this is very hard to read

Reply:
Yes, but no other solution works at the moment.